src/NovelAudioService.cpp

```diff
@@ -4,33 +4,20 @@
 #include "NovelRTUtilities.h"
 #include <iostream>
 #include <functional>
+#include <stdexcept>
 
 namespace NovelRT {
 
-NovelAudioService::NovelAudioService() : _nextChannel(1), _musicTime(0), _musicPausedTime(0), isInitialized(false) {
+NovelAudioService::NovelAudioService() : _nextChannel(1), _musicTime(0), _musicPausedTime(0), isInitialized(false), _logger(NovelUtilities::CONSOLE_LOG_AUDIO) {
```

Comment:
Shouldn't this be `NovelRTUtilities`?

Reply:
The filename says `NovelRTUtilities`, however the class was renamed to `NovelUtilities` during the first audio merge (iirc). If you want, I can update the filename as well (shouldn't be a hassle)

Reply:
Eh, the filename will be reviewed and changed - pending UBO merge. I want to be sure that changing the file name doesn't ruin anything for that feature, and it'd be easier for this to merge afterwards